src/content/docs/zh-cn/distribute/rpm.mdx

```diff
@@ -0,0 +1,339 @@
+---
+title: RPM
+sidebar:
+  order: 1
+---
+
+import CommandTabs from '@components/CommandTabs.astro';
+
+# 打包为 RPM
+
+:::note
+这个指南中的一些部分是可选的。这包括配置脚本和其他特定步骤。根据您的具体需求和要求，可以自由地调整说明。
+:::
+
+本指南介绍了如何分发和管理RPM软件包，包括获取软件包信息、配置脚本、设置依赖关系和签名软件包。
+
+## RPM 包配置
+
+Tauri 允许您通过添加脚本、设置依赖项、添加许可证、包含自定义文件等来配置 RPM 软件包。
+有关可配置选项的详细信息，请参考：[RpmConfig](https://v2.tauri.app/reference/config/#rpmconfig)。
+
+### 添加 post、pre-install/remove 脚本到安装包
+
+RPM 软件包管理器允许您在安装或删除软件包之前或之后运行脚本。例如，您可以使用这些脚本在软件包安装后启动一个服务。
+
+下面是添加这些脚本的示例：
+
+1. 在项目的 `src-tauri` 目录下创建一个名为 `scripts` 的文件夹。
+
+```bash
+mkdir src-tauri/scripts
+```
+
+2. 在文件夹中创建脚本文件。
+
+```bash
+touch src-tauri/scripts/postinstall.sh \
+touch src-tauri/scripts/preinstall.sh \
+touch src-tauri/scripts/preremove.sh \
+touch src-tauri/scripts/postremove.sh
+```
+
+现在如果我们查看 `/src-tauri/scripts`，我们将看到。
+
+```bash
+ls src-tauri/scripts/
+postinstall.sh  postremove.sh  preinstall.sh  preremove.sh
+```
+
+3. 向脚本添加一些内容。
+
+```bash title="preinstall.sh"
+echo "-------------"
+echo "This is pre"
+echo "Install Value: $1"
+echo "Upgrade Value: $1"
+echo "Uninstall Value: $1"
+echo "-------------"
+```
+
+```bash title="postinstall.sh"
+echo "-------------"
+echo "This is post"
+echo "Install Value: $1"
+echo "Upgrade Value: $1"
+echo "Uninstall Value: $1"
+echo "-------------"
+```
+
+```bash title="preremove.sh"
+echo "-------------"
+echo "This is preun"
+echo "Install Value: $1"
+echo "Upgrade Value: $1"
+echo "Uninstall Value: $1"
+echo "-------------"
+```
+
+```bash title="postremove.sh"
+echo "-------------"
+echo "This is postun"
+echo "Install Value: $1"
+echo "Upgrade Value: $1"
+echo "Uninstall Value: $1"
+echo "-------------"
+```
+
+4. 将脚本添加到 `tauri.conf.json` 文件中
+
+```json title="tauri.conf.json"
+{
+  "bundle": {
+    "linux": {
+      "rpm": {
+        "epoch": 0,
+        "files": {},
+        "release": "1",
+        // add the script here
```

Comment:
```suggestion
        // 在此添加脚本
```
Generally I think it's okay to not translate comment because it can be improved later, but since this particular part is tied to the instruction I think it needs to be translated out now.